src/tests/ftest/control/dmg_scale.py

```diff
@@ -0,0 +1,185 @@
+    "engine_pool_scrubber_corruption_total",
+    "engine_pool_scrubber_csums_current",
+    "engine_pool_scrubber_csums_prev",
+    "engine_pool_scrubber_csums_total",
+    "engine_pool_scrubber_next_csum_scrub",
+    "engine_pool_scrubber_next_tree_scrub",
+    "engine_pool_scrubber_prev_duration",
+    "engine_pool_scrubber_prev_duration_max",
+    "engine_pool_scrubber_prev_duration_mean",
+    "engine_pool_scrubber_prev_duration_min",
+    "engine_pool_scrubber_prev_duration_stddev",
+    "engine_pool_scrubber_scrubber_started",
+    "engine_pool_scrubber_scrubs_completed",
+    "engine_pool_started_at",
+    "engine_pool_vos_aggregation_akey_deleted",
+    "engine_pool_vos_aggregation_akey_scanned",
+    "engine_pool_vos_aggregation_akey_skipped",
+    "engine_pool_vos_aggregation_csum_errors",
+    "engine_pool_vos_aggregation_deleted_ev",
+    "engine_pool_vos_aggregation_deleted_sv",
+    "engine_pool_vos_aggregation_dkey_deleted",
+    "engine_pool_vos_aggregation_dkey_scanned",
+    "engine_pool_vos_aggregation_dkey_skipped",
+    "engine_pool_vos_aggregation_epr_duration",
+    "engine_pool_vos_aggregation_epr_duration_max",
+    "engine_pool_vos_aggregation_epr_duration_mean",
+    "engine_pool_vos_aggregation_epr_duration_min",
+    "engine_pool_vos_aggregation_epr_duration_stddev",
+    "engine_pool_vos_aggregation_merged_recs",
+    "engine_pool_vos_aggregation_merged_size",
+    "engine_pool_vos_aggregation_obj_deleted",
+    "engine_pool_vos_aggregation_obj_scanned",
+    "engine_pool_vos_aggregation_obj_skipped",
+    "engine_pool_vos_aggregation_uncommitted",
+    "engine_pool_vos_space_nvme_used",
+    "engine_pool_vos_space_scm_used",
+    "engine_pool_xferred_fetch",
+    "engine_pool_xferred_update",
+    "engine_pool_EC_update_full_stripe",
+    "engine_pool_EC_update_partial",
+    "engine_pool_block_allocator_alloc_hint",
+    "engine_pool_block_allocator_alloc_large",
+    "engine_pool_block_allocator_alloc_small",
+    "engine_pool_block_allocator_frags_aging",
+    "engine_pool_block_allocator_frags_large",
+    "engine_pool_block_allocator_frags_small",
+    "engine_pool_block_allocator_free_blks",
+    "engine_pool_ops_key2anchor"
+]
+
+
+class DmgScale(TestWithServers):
+    """Verify dmg commands works as expected in a large scale system.
+
+    :avocado: recursive
+    """
+
+    def test_dmg_scale(self):
+        """Run the following steps at Aurora and manually collect duration for each step.
+
+        0. Format storage
+        1. System query
+        2. Create a 100% pool that spans all engines
+        3. Pool query
+        4. Pool destroy
+        5. Create 49 pools spanning all the engines with each pool using a 1/50th of the capacity
+        6. Pool list
+        7. Query around 80 pool metrics
+        8. Destroy all 49 pools
+        9. System stop
+        10. System start
+
+        Jira ID: DAOS-10508.
+
+        :avocado: tags=all,manual
+        :avocado: tags=hw,medium
```

Comment:
If manual, we should remove hw,medium
```suggestion
        :avocado: tags=all,manual
```

Reply:
Done